src/Serialization/Fields.jl

```diff
@@ -114,10 +116,12 @@ end
 #elements
 @registerSerializationType(fqPolyRepFieldElem)
 @registerSerializationType(nf_elem)
-
 @registerSerializationType(Hecke.NfRelElem)
+@registerSerializationType(FqFieldElem)
+
+FieldElemTypes = Union{nf_elem, fqPolyRepFieldElem, Hecke.NfRelElem, FqFieldElem}
```

Comment:
```suggestion
const FieldElemTypes = Union{nf_elem, fqPolyRepFieldElem, Hecke.NfRelElem, FqFieldElem}
```
Otherwise this leads to serious type instability

Reply:
Also, maybe the name is a bit too generic, someone might end up using it accidentally? Dunno *shrug*